config/locales/en.yml

```diff
@@ -1031,7 +1031,7 @@ en:
           orchard: "Orchard"
           plant_nursery: "Plant Nursery"
           quarry: "Quarry"
-          railway: "Railway"
+          railway: "Railroad Right of Way"
```

Comment:
Wiki says "an area of land dedicated to train operation or support, around tracks, yards, sidings, station complexes, and ancillary man-made objects along the tracks", and I've frequently seen this used in the latter meanings - land owned by the railway but not necessarily with tracks on it. Could be a locomotive shed too. It seems to be that your "right of way" term is not covering that.

Reply:
On second thought, we can probably get away with calling it “Railroad Corridor”, since the broader meaning (say, everything within 10&nbsp;miles of the tracks) would be unmappable anyways.